doremus.ttl

```diff
@@ -1601,6 +1601,40 @@ Examples:"""@en ;
                        rdfs:label "M9 Derivation Type"@en ,
                                   "M9 Type de dérivation"@fr .
 
+###  http://data.doremus.org/ontology#U83_has_relative_key
+mus:U83_has_relative_key rdf:type owl:SymmetricObjectProperty ;
+    rdfs:domain mus:M4_Key ;
+    rdfs:range mus:M4_Key ;
+    rdfs:isDefinedBy <http://data.doremus.org/ontology#> ;
+    rdfs:label "U83 has relative key"@en , "U83 a pour tonalité relative"@fr ;
+    rdfs:comment "Scope Note: This class allows to connect two M4 Keys that are in a relative relationship, i.e. one is the relative major or minor of the other one."@en ,
+    "Scope Note: Cette classe permet de connecter deux M4 Tonalité qui sont en relation par rapport à l'armure, c'est-à-dire l'une est la relatif majeur ou mineur de l'autre."@fr ,
```

Comment:
Line 1611: s/l'armure/l'autre

Reply:
Sorry, I am not getting, can you make this change request more explicit please?

Reply:
Replace 'par rapport à l'armure' by 'par rapport à l'autre' in the French comment

Reply:
Ah, I wanted to say exactly armure. See [wikipedia](https://fr.wikipedia.org/wiki/Tonalité_relative) : 
> ... qu'une même armure est partagée entre une tonalité majeure et une tonalité mineure

The point: I do not know how to make "relative relationship" in French. I would say "relation relative" but it is not sounds correct. So I preffered to specify that they are relatives in the sense of the key signature ("armure").

Reply:
Ah, my bad, or more precisely my ignorance of the proper definition of this concept. Your description is good!